src/app/components/CourseChoiceButton/CourseChoiceButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactNode } from "react";
+import styles from "./CourseChoiceButton.module.css";
+
+type CourseChoiceButtonProps = {
+  children: ReactNode;
+};
+
+function CourseChoiceButton({
```

Comment:
What makes this button so special that it is called `CourseChoiceButton`?
For me, it looks like a normal button.
You can use it like this too:
`<CourseChoiceButton>Submit</CourseChoiceButton>`

Reply:
Thanks. Done!